modules/gumgumBidAdapter.js

```diff
@@ -222,7 +222,29 @@ function _getFloor (mediaTypes, staticBidFloor, bid) {
   return bidFloor;
 }
 
-function getEids (userId) {
+/**
+ * loops through bannerSizes array to get greatest slot dimensions
+ * @param {number[][]} sizes
+ * @returns {number[]}
+ */
+function getGreatestDimensions(sizes) {
+  let maxw = 0;
+  let maxh = 0;
+  let greatestVal = 0;
+  sizes.forEach(bannerSize => {
+    let [width, height] = bannerSize;
+    let greaterSide = width > height ? width : height;
+    if ((greaterSide > greatestVal) || (greaterSide === greatestVal && width >= maxw && height >= maxh)) {
```

Comment:
Just to confirm, "greatest dimensions" is defined here as the size with the single largest side, not the largest area.

E.g. given the following sizes:
`[[1001,1],[1000,1000]]`
you expect to return:
`[1001,1]`

Is that correct?

Reply:
hi @gwhigs - yes, we'd expect [1001,1] returned